config/config.go

```diff
@@ -63,9 +63,32 @@ func FindConfigFile(start string) (string, error) {
 	return "", fmt.Errorf("Config file not found")
 }
 
+type DotenvStoreConfig struct{}
+
+type INIStoreConfig struct{}
+
+type JSONStoreConfig struct{
+	Indent int `yaml:"indent"`
+}
+
+type JSONBinaryStoreConfig struct{}
```

Comment:
Shouldn't this also have an `indent` setting?

Reply:
Yes, you are right, I will add this parameter

Reply:
Here it is, I implemented it this way so we can have a different configuration for `json` and `json_binary`. Let me know what you think.